### DISCUSSION TITLE
Show total of connected peers in the monitor screen?

### DISCUSSION THREAD

**Discussion by lilws** [jesec/flood#263] [Q&A]
In rtorrent cli, I can see that info show up in the right bottom screen. Is it possible for flood to implement that feature? Show number of connected peers / total peers rtorrent allow? Of course one for dowload and the other for upload.

<img width="160" alt="Screen Shot 2021-04-02 at 07 39 51" src="https://user-images.githubusercontent.com/12478478/113367641-c2eaca80-9386-11eb-8776-af87efe1379c.png">
<img width="176" alt="Screen Shot 2021-04-02 at 07 39 26" src="https://user-images.githubusercontent.com/12478478/113367644-c4b48e00-9386-11eb-831c-25e0efb2806b.png">

![rt](https://user-images.githubusercontent.com/12478478/113367724-ffb6c180-9386-11eb-88ae-e94df6f0e03a.png)

**Comment by jesec** [comment]
Good idea. But we have to find a way to fit that into the design. Can you share some potential ways (where should we place it, etc.), if possible?

**Reply by lilws** [reply]
Thanks. I'm thinking of either move it below the total uploaded and downloaded or in the middle of the sorted and start/pause main bar. It won't ruin the flood UI.
